index.js

```diff
@@ -49,7 +70,25 @@ getUsersCreditDetails(usersArray);
 // ***************************************************************************
 
 const genderView = users => {
-  // Your code goes here ...
+  const females = users.filter(function(user) {
+    return user.gender === 'female';
+  })
+
+  const males = users.filter(function(user) {
+    return user.gender === 'male';
+  })
+
+  const femaleUsers = [];
+  const maleUsers = [];
+
+  for (let user of females) {
+    femaleUsers.push(`${user.firstName} ${user.lastName}`);
+  }
+  for (let user of males) {
+    maleUsers.push(`${user.firstName} ${user.lastName}`);
+  }
+
+  return {femaleUsers: femaleUsers, maleUsers: maleUsers};
```

Comment:
in ES6 the object shorthand was also introduced :)

```
return {
   femaleUsers,
   maleUsers
}
```

No need for the repetition